docs/aks/README.md

```diff
@@ -392,63 +430,75 @@ An helm chart is included for the reference application ([helium](https://github
 
 Install the Helm Chart located in the cloned directory
 
-```shell
-cd $REPO_ROOT/docs/aks/cluster/charts
+```bash
+
+cd $REPO_ROOT/docs/aks/cluster/charts/helium
+
 ```
 
-A file called helm-config.yaml with the following contents that needs be edited to fit the environment being deployed in. The file looks like this
+A file called helm-config.yaml with the following contents that needs be to edited to fit the environment being deployed in. The file looks like this
 
 ```yaml
 # Default values for helium.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 labels:
-  aadpodidbinding: %%MSI_Name%% #should be value of $MSI_Name from the output of aad-podid.sh
+  aadpodidbinding: %%MSI_Name%% # Should be value of $MSI_Name from the output of aad-podid.sh
 
 image:
-  repository: retaildevcrew #The spceific acr created for this environment
+  repository: retaildevcrew # The spceific acr created for this environment
   name: helium-csharp # The name of the image for the helium-csharp repo
 
 annotations:
   linkerd.io/inject: enabled # Allows for the application to be injected into the Service Mesh
 
 ingress:
   hosts:
-    - host: %%INGRESS_PIP%%.nip.io # Replace the IP address with the IP of the nginx external IP. kubectl get svc -n ingress-nginx to see the correct IP
+    - host: %%INGRESS_PIP%%.nip.io # Replace the IP address with the IP of the nginx external IP (value of $INGRESS_PIP or run kubectl get svc -n ingress-nginx to see the correct IP)
       paths: /
 
-keyVaultName: %%KV_Name%% # Replace with the name of the Key Vault that holds the secrets
+keyVaultName: %%KV_Name%% # Replace with the name of the Key Vault that holds the secrets (value of $He_Name)
 ```
 
 Replace the values in the file surrounded by `%%` with the proper environment variables
 
-```shell
+```bash
+
 sed -i "s/%%MSI_Name%%/${MSI_Name}/g" helm-config.yaml && \
 sed -i "s/%%INGRESS_PIP%%/${INGRESS_PIP}/g" helm-config.yaml && \
 sed -i "s/%%KV_Name%%/${He_Name}/g" helm-config.yaml
+
 ```
 
 This file can now be given to the the helm install as an override to the default values.
 
-```shell
-helm install helium-aks helium -f helm-config.yaml
+```bash
+
+cd $REPO_ROOT/docs/aks/cluster/charts
+
+helm install helium-aks helium -f ./helium/helm-config.yaml
+
```

Comment:
@evillgenius75 -- I had to update this as the helm-config.yaml file is in the helium folder, let me know if it's preferred to move the yaml file up a directory.